### DISCUSSION TITLE
Unable to figure out reason for The request is rejected, please check the consumer_name for this request

### DISCUSSION THREAD

**Discussion by ms9580** [apache/apisix#10184] [Q&A]
Following is a route definition.

{
  "uri": "/ivsapp/bioapp/*",
  "name": "ivs-php-services-route",
  "methods": [
    "GET",
    "POST",
    "PUT",
    "DELETE",
    "PATCH",
    "HEAD",
    "OPTIONS",
    "CONNECT",
    "TRACE",
    "PURGE"
  ],
  "vars": [
    [
      "server_port",
      "==",
      "9783"
    ]
  ],
  "plugins": {
    "consumer-restriction": {
      "_meta": {
        "disable": false
      },
      "whitelist": [
        "ivsusersfe"
      ]
    },
    "proxy-rewrite": {
      "scheme": "https"
    },
    "response-rewrite": {
      "_meta": {
        "disable": true
      },
      "headers": {
        "remove": [
          "X-Userinfo",
          "X-Access-Token",
          "Cookie",
          "Authorization"
        ]
      }
    }
  },
  "service_id": "477653067647423716",
  "upstream_id": "477652852714509540",
  "labels": {
    "API_VERSION": "1.0.0"
  },
  "status": 1
}

The consumer being referred in consumer_restriction plugin is:

{
  "username": "ivsusersfe",
  "plugins": {
    "openid-connect": {
      "_meta": {
        "disable": false
      },
      "bearer_only": true,
      "client_id": "ivs-apisix",
      "client_secret": "abc1234",
      "discovery": "https://ssosit..com:8183/auth/realms/ivs/.well-known/openid-configuration",
      "introspection_cache_ignore": true,
      "introspection_endpoint": "https://ssosit.com:8183/auth/realms/ivs/protocol/openid-connect/token/introspect",
      "introspection_endpoint_auth_method": "client_secret_basic",
      "realm": "ivs"
    }
  }
}

When calling the API using postman, I am receiving the following as response and I am not able to figure the reason. Any help is appreciated.

{"message":"The request is rejected, please check the consumer_name for this request"}

Thank you.

Regards.

**Comment by Bishnup1995** [comment]
@ms9580 Any luck with the solution?
